oemer/build_label.py

```diff
@@ -107,6 +107,6 @@ def find_example(dataset_path: str, color: int, max_count=100, mark_value=200):
     #out = build_label(path)
 
     color = 45
-    arr = find_example(color)
+    arr = find_example(color) # type: ignore
```

Comment:
There should be two spaces before the commented message. Please also modify the other places that have the same problem.